Write up  process of nominating maintainer (#206)
astropy/astropy-project: messages/maintainer_access.md

Hello {NAME},

> 

Reviewer: Needs an intend at the top. This is one possible place that new maintainers might come to when they want to know how they become a maintainer, who can nominate them, if they can self-nominate. As discussed in the Coco, this is not the place we intend to lay this out. So, this needs a statement of intend and a point to where to find that info on the top.

Author: should be in the commit I'm about to push up

> You have been nominated as {role} and the Coordinating Committee enthusiastically accepted this nomination! We would be thrilled if you are able to accept this nomination. {More context might be necessary for some people - fill it in here}

Reviewer 2: ```suggestion
You have been nominated as {role} and the Coordinating Committee has enthusiastically accepted this nomination! We would be thrilled if you are able to accept this nomination. {More context might be necessary for some people - fill it in here}
```

> You can see the formal description of the maintainers role at http://www.astropy.org/team.html. If you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project, or if absolutely necessary, the Coordination Committee.

Reviewer 3: Since this is the first communication with the person nominated to be a maintainer and they are expected to answer whether they accept the nomination, would it make sense to elaborate a little more about the expectations from a maintainer? I know that this might be hard to frame since it is a "voluntary" task, but it might help make the decision. I checked http://www.astropy.org/team.html -- it just lists the tasks a maintainer is supposed to do without any concrete ideas about time commitment. This is the first thing that came to my mind when I thought about what I would wonder about if I received such an email/correspondence, but I don't know if there is a specific answer to this!

If you would like to accept this nomination, we ask that you familiarize yourself with the following Project guidelines:

* [Developer guidelines for Astropy](https://docs.astropy.org/en/latest/#developer-documentation)
* [Basic development workflow](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html)
* [Guidelines on when to squash or rebase](https://docs.astropy.org/en/latest/development/when_to_rebase.html)
* [Astropy Project Code of Conduct](https://www.astropy.org/code_of_conduct.html)

> 

Reviewer 2: We discussed on the CoCo call that it might also make sense to add a statement to encourage nominees / new maintainers to join the monthly dev telecons. What do you think @eteq?

Author: Agreed, added in commit I'm about to push up.

Hello {NAME},

You have been nominated as {role} and the Coordinating Committee has enthusiastically accepted this nomination! We would be thrilled if you are able to accept this nomination. {More context might be necessary for some people - fill it in here}

> In case you are not familiar with the role, maintainers are the people who keep a package or sub-package working. In particular, they review and merge PRs, keep track of what's going on and organize ideas for future development (see https://www.astropy.org/team.html#Subpackage_maintainer for details). In practice this means a fair amount of community interaction when reviewing PRs and related discussion.  As a general guideline, if you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project.  If necessary, you should also real free to reach out to the Coordination Committee (by @-mention on github or via coordinators@astropy.org), as the Coordination Committee has the authority to make decisions when consensus cannot be acheived.

Reviewer: ```suggestion
In case you are not familiar with the role, maintainers are the people who keep a package or sub-package working. In particular, they review and merge PRs, keep track of what's going on and organize ideas for future development (see https://www.astropy.org/team.html#Subpackage_maintainer for details). In practice this means a fair amount of community interaction when reviewing PRs and related discussion.  As a general guideline, if you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project.  If necessary, you should also free to reach out to the Coordination Committee (by @-mention on github or via coordinators@astropy.org), as the Coordination Committee has the authority to make decisions when consensus cannot be achieved.
```

If you would like to accept this nomination, we ask that you familiarize yourself with the following Project guidelines:

* [Developer guidelines for Astropy](https://docs.astropy.org/en/latest/#developer-documentation)
* [Basic development workflow](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html)
* [Guidelines on when to squash or rebase](https://docs.astropy.org/en/latest/development/when_to_rebase.html)
* [Astropy Project Code of Conduct](https://www.astropy.org/code_of_conduct.html)

> If you accept, please respond indicating that you've reviewed (or plan to review) the developer documentation and that you have read and agree to abide by the Code of Conduct. After we receive that email from you, we will make an announcement and there will be a two week period of comment. After that period is over, your name will be added to the team webpage and you will be given the appropriate Github permissions.

Reviewer 4: ```suggestion
If you accept, please respond indicating that you've reviewed the developer documentation and that you have read and agree to abide by the Code of Conduct. After we receive that email from you, we will make an announcement and there will be a two week period of comment. After that period is over, your name will be added to the team webpage and you will be given the appropriate Github permissions.
```
